### DISCUSSION TITLE
Critical Vulnerability in System.Drawing.Common v5.0.0

### DISCUSSION THREAD

**Discussion by abhjitsantra** [elsa-workflows/elsa-core#4359] [Design & tech]
`System.Drawing.Common v5.0.0` nupkg has one critical vulnerability. `System.Drawing.Common v5.0.0` package is being indirectly referenced by `DistributedLock v2.3.2`. Can we update the library `Elsa.Workflows.Runtime` to reference `DistributedLock v2.3.3` which indirectly references `System.Drawing.Common v7.0.0`?

**Comment by sfmskywalker** [comment]
Hi @abhjitsantra , thanks for the head-sup!
We should definitely upgrade the packages. Would you like to submit a PR?

**Reply by abhjitsantra** [reply]
Here is the PR for it - #4360 @sfmskywalker .

**Comment by abhjitsantra** [comment]
The package has been upgraded.
